Assets/Scripts/Contract/Contract.cs

```diff
@@ -20,11 +20,6 @@ public Contract(string contractName, float totalEffort, List<string> assignedWor
     {
         this.contractName = contractName;
         this.totalEffort = totalEffort;
-        // TODO: This needs to be addressed. Workers will not be assigned to contracts when they are created so we do not need this.
```

Comment:
If this change was made, I would also suggest removing the entry for assigned workers in the constructor method as to not confuse the person using it.